src/app/services/directly-follows-graph/svg.service.ts

```diff
@@ -239,10 +239,10 @@ export class SvgService {
     }
 
     private setCoordinates(edge: Edge, graph: Graph) {
-        let startX: number = 0;
-        let startY: number = 0;
-        let endX: number = 0;
-        let endY: number = 0;
+        let startX: number;
```

Comment:
Ist diese Änderung vom Linter? In jedem Fall frage ich mich, warum das für die ersten vier Variablen geändert worden ist, aber nicht für die nächsten vier?

Reply:
Das war eine der Webstorm CleanUps, die ich mal durchgeführt habe. 
Der Grund dafür ist, dass die anderen Variablen unter Umständen die 0 beibehalten, die veränderte jedoch immer zu einem späteren Zeitpunkt auf einen neuen Wert gesetzt wird.